Building an API.md

```diff
@@ -0,0 +1,53 @@
+# Intro
+
+Building an API is different than building a regular HTML application. There are
+no HTML views to worry about, but instead you should take extra care about status responses, response format and especially documentation since your application will be used
+by other developers.
+
+## Creating an application
+
+To generate a Rails application with only the elements necessary for an API
+application use the `--api` flag e.g.
+`rails new my_application --api`
+
+We write our APIs in JSON format, more specifically using the [JSON API standard](http://jsonapi.org/).
+This allows those who consume our APIs to know what to expect regarding the
```

Comment:
I love the fact that you added reasoning to why we use the JSON API standard. 
Please also note that the reason we use the standard is that the devs building the client apps can use libraries that are built to consume the JSON API standard, making it easier and faster to build their clients.